driver-core/src/main/com/mongodb/TaggableReadPreference.java

```diff
@@ -52,17 +53,20 @@
 
     private final List<TagSet> tagSetList = new ArrayList<TagSet>();
     private final Long maxStalenessMS;
+    private boolean hedgedReads;
```

Comment:
The end result of this work is to send this value to the server in a `hedge` document.  To do that, we have to add something to the `toDocument` method below.

The spec also requires that the `hedge` document is not sent unless the application explicitly configures hedging, so this field will have to be nullable (easy if you add an options class as requested above).

Reply:
There should be an integration test to ensure that what we send to the server is accepted by the server.

Reply:
I have added an integration test in `FindOperationSpecification`.